variants/generator/empiresandcoalitions.yml

```diff
@@ -0,0 +1,53 @@
+# The first year of the game
+START_YEAR: 1800
+# The starting units
+START_UNITS:
+  Austria:
+    Army: ['Vienna', 'Venice', 'Budapest']
+    Fleet: []
+  Britain:
+    Army: ['Hanover']
+    Fleet: ['Edinburgh', 'London', 'Gibraltar']
+  Denmark:
+    Army: []
+    Fleet: ['Copenhagen', 'Christiania']
+  France:
+    Army: ['Paris', 'Brest', 'Lyon']
+    Fleet: ['Marseilles']
+  Prussia:
+    Army: ['Berlin', 'Konigsberg', 'Breslau']
+    Fleet: []
+  Sicily:
+    Army: ['Napels']
+    Fleet: ['Palermo']
+  Spain:
+    Army: []
+    Fleet: ['Madrid', 'Valencia']
+  Russia:
+    Army: ['Moscow', 'Kiev']
+    Fleet: ['St. Petersburg', 'Sevastopol']
+  Turkey:
```

Comment:
This doesn't affect anything - but I originally generated the variant using "Turkey" rather than "Ottoman Empire".